Doc/library/string.rst

```diff
@@ -510,10 +510,12 @@ The available presentation types for :class:`float` and
    | ``'F'`` | Fixed-point notation. Same as ``'f'``, but converts      |
    |         | ``nan`` to  ``NAN`` and ``inf`` to ``INF``.              |
    +---------+----------------------------------------------------------+
-   | ``'g'`` | General format.  For a given precision ``p >= 1``,       |
+   | ``'g'`` | General format. For a given precision ``p >= 1``,        |
```

Comment:
The change in this line was a bit gratuitous. Happy to revert.

Reply:
I personally don't care about double spaces between sequences from a readability point of view (but I think some people care, I mean, the [double space was added on purpose](https://en.wikipedia.org/wiki/Sentence_spacing)).

However, this diff will flag the sentence as *fuzzy* (to review) in all documentation translations, so it'll cost time to ~8 persons. From this point of view, it's better to revert it.

Reply:
Ah, good point. Thanks, @JulienPalard. Now fixed.